storybookjs/storybook
# Extract Jest config from package.json to config file

Issue: https://github.com/storybooks/storybook/issues/2407

## What I did
 - Extract Jest config of _cra-kitchen-sink_ example  from _package.json_ to an external config file so that CRA is happy and let us run the test via `yarn run test`.
The idea is to have both **test** & **generate-addon-jest-testresults** scripts to work with no interactions/limitations.

## How to test
- Checkout all-the-things
- Go to _cra-kitchen-sink_ example project
- run `yarn run test` - it should run smoothly with some "expected" failure.
- then run `yarn run generate-addon-jest-testresults` it should generate the JSON results.

Is this testable with jest or storyshots?
Nope

Does this need a new example in the kitchen sink apps?
Nope

Does this need an update to the documentation?
Nope

If your answer is yes to any of these, please make sure to include it in your PR.
